app/presenters/summary/sections/base_section.rb

```diff
@@ -29,7 +29,7 @@ def title
         I18n.t(
           name,
           scope: 'summary.sections',
-          subject: I18n.t('summary.dictionary.subject', subject_type:),
+          subject: I18n.t("summary.dictionary.subjects.#{subject_type}"),
```

Comment:
@mseedat-moj is this change necessary? This is handled here: 
https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/blob/main/config/locales/en.rb

and is used elsewhere I think.

Reply:
@mseedat-moj forgive me, I see this is for the summary dictionary.